assembly/src/ast/instruction/print.rs

```diff
@@ -12,9 +12,9 @@ impl PrettyPrint for Instruction {
         match self {
             Self::Nop => const_text("nop"),
             Self::Assert => const_text("assert"),
-            Self::AssertWithError(err_code) => {
-                flatten(const_text("assert.err") + const_text("=") + display(err_code))
```

Comment:
Looks good - except could you apply this same fix to all other variants that use `display()` like this?

That + fixing the CI and we should be good to merge